pkgs/tools/inputmethods/hime/default.nix

```diff
@@ -1,27 +1,44 @@
-{
-stdenv, fetchFromGitHub, pkg-config, which, gtk2, gtk3, qt4, qt5, libXtst, lib,
+{ stdenv
+, fetchFromGitHub
+, pkg-config
+, which
+, gtk2
+, gtk3
+, qt4
+, qt5
+, libXtst
+, lib
+, libchewing
+, unixtools
+, anthy
+, enableChewing ? true
+, enableAnthy ? true
```

Comment:
Do we really need those options?

Reply:
Thanks for reviewing this, and provide me the suggestion.

Chewing only used in Taiwan, and Anthy is for Japan.

Because the setting panel of the program is not hide the setting for Chewing and Anthy even there are disabled, so it is good to make these options are opt-in.  

Generally, there is big combination tables for the Asia input methods, such that people who really care about what they install, may want to remove this.

It is both good for me to remove or keep these options, because I use Array most and Chewing a little and do not care about the storage of tables in my laptop.